starlette_admin/translations/fr/LC_MESSAGES/admin.po

```diff
@@ -157,10 +157,20 @@ msgstr "Connectez-vous à votre compte"
 msgid "Username"
 msgstr "Nom d'utilisateur"
 
+#: starlette_admin/templates/login.html:36
+#, fuzzy
+msgid "Enter username"
+msgstr "Nom d'utilisateur"
+
 #: starlette_admin/templates/login.html:43
 msgid "Password"
 msgstr "Mot de passe"
 
+#: starlette_admin/templates/login.html:47
+#, fuzzy
+msgid "Enter password"
+msgstr "Mot de passe"
```

Comment:
```suggestion
msgstr "Entrez le mot de passe"
```

Reply:
These were automatically created by i18n using fuzzy. Can you also remove the `#, fuzzy`?